source/backend/api/Services/CompensationRequisitionService.cs

```diff
@@ -59,8 +62,15 @@ public PimsCompensationRequisition Update(PimsCompensationRequisition compensati
 
             var currentCompensation = _compensationRequisitionRepository.GetById(compensationRequisition.CompensationRequisitionId);
 
-            CheckDraftStatusUpdateAuthorized(currentCompensation.IsDraft, compensationRequisition.IsDraft);
-            CheckTotalAllowableCompensation(compensationRequisition.AcquisitionFileId, compensationRequisition);
+            var currentAcquisitionFile = _acqFileRepository.GetById(currentCompensation.AcquisitionFileId);
+            var currentAcquisitionStatus = Enum.Parse<AcqusitionStatusTypes>(currentAcquisitionFile.AcquisitionFileStatusTypeCode);
+
+            if (!_statusSolver.CanEditOrDeleteCompensation(currentAcquisitionStatus, currentCompensation.IsDraft) && !_user.HasPermission(Permissions.SystemAdmin))
```

Comment:
I actually don't see a requirement for an admin to be able to override this in the story, can you point that out to me?

Reply:
It's from a different story. I figured those where additive instead of replaced.